### PR TITLE
Update GraphQL libraries and parsers

### PR DESCRIPTION
**Pull request by dotansimha**
This PR includes changes to the list of ESLint tools related to GraphQL. 
 
- The new library works as a parser for ESLint, by converting the GraphQL AST to ESTree structure.
- Supports schema and operations validation.
- More details: https://the-guild.dev/blog/introducing-graphql-eslint

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -150,7 +150,9 @@ If you want to contribute, please read the [contribution guidelines](contributin
 
 ### Libraries
 
-- [GraphQL](https://github.com/apollostack/eslint-plugin-graphql) - Check your GraphQL query strings against a schema.
+- GraphQL
+  - [`dotansimha/graphql-eslint`](https://github.com/dotansimha/graphql-eslint) - Validates, prettifies and checks your GraphQL operations and GraphQL schema for best-practices.
```
Comment (dustinspecker) [inline readme.md:154]: Can you remove the backticks on both of these, please?

**Comment by dustinspecker** [comment]
Thank you! Looks neat! Left a comment and then happy to merge.

File: readme.md (hunk 1 above)
Comment (dotansimha) [reply readme.md]: Done!

**Comment by dustinspecker** [comment]
Thank you!

**Pull request merged**